docs/meshstack.index.md

```diff
@@ -36,6 +36,6 @@ The connections are shown in the following matrix table:
 | Cloud Foundry |              Organization               |                 Space                 |              UAA User              |                    Quota                    |
```

Comment:
d: Actually even in OpenStack it is optional to map the customer to a domain. At VW we have a flat domain model in place. all projects are located in the default domain. This is something that can be configured. So you can either represent a customer as a domain, or the customer-identifier will be part of the project-name

Reply:
can we put (optional) in brackets next to the mapping to make this transparent?

Reply:
Would it fit in the overview page https://docs.meshcloud.io/docs/meshstack.index.html? I think we are trying to give a high level idea about the mappings. It also says `For more details, please consult documentation of the platforms.`

Reply:
I think it makes sense to mention that it is not always reflected as a domain. It depends on the configuration. But details about this should be documented in a separate page. You'd have to figure out where it fits best

Reply:
Okay. I will do it here, but I will not try to update any OpenStack related page right now because my goal with this PR was to fix this small inconsistency.